ui/app/scripts/controllers/blockersCtrl.js

```diff
@@ -172,6 +180,23 @@ angular.module('apollo')
           $scope.allServices = tempServices;
       });
 
+      apolloApiService.getAllStacks().then(function(response) {
+        response.data.forEach(function(stack) {
+            $scope.allStacks.push(stack);
+        });
+      });
+
+      $scope.getStack = function(stackId) {
+        if ($scope.allStacks.length === 0) {
+            return;
+        }
+        $scope.allStacks.find(function(stack) {
+           if (stack.id === stackId) {
+                $scope.blockersStacks[stackId] = stack;
+           };
+        });
+      };
```

Comment:
!$scope.allStacks.length && $scope.allStacks.find(function(stack) {
           if (stack.id === stackId) {
                $scope.blockersStacks[stackId] = stack;
           };

Reply:
Probably this function will be removed all together